### ISSUE TITLE
Docs request: getting started for Python users

### ISSUE THREAD

**Issue by jamesbraza** [oguimbal/pg-mem#367]
I am looking into using pg-mem for some integration testing of my FastAPI + SQL Alchemy code.

It looks like the majority of this uses TypeScript, which obviously is not Python.

This request is just make some quick instructions somewhere for Python users, to be able to stand this up for use within a `pytest` suite

**Comment by Lewenhaupt** [comment]
Yeah I doubt that's a doc-fix. This lib is written to work in the browser and in Node.
